lightly/api/api_workflow_datasources.py

```diff
@@ -216,6 +194,12 @@ def download_raw_predictions(
                 "'run_id' should not be `None` when 'relevant_filenames_artifact_id' "
                 "is specified."
             )
+        relevant_filenames_kwargs = {}
+        if run_id and relevant_filenames_artifact_id:
```

Comment:
```suggestion
        if run_id is not None and relevant_filenames_artifact_id is not None:
```

Would be better to also explicitly check for `None` in the checks above.

Reply:
Updated